### PR TITLE
Load variables from json file

### PR DESCRIPTION
**Pull request by mithunbharadwaj**

### CODE REVIEW

File: asabiris/formatter/jinja/service.py [hunk 1]
```diff
@@ -1,39 +1,75 @@
...
-		jinja_variables = collections.ChainMap(template_params, self.Variables)
-		template_io = await self.App.LibraryService.read(template_path)
-		if template_io is None:
-			raise PathError("Template '{}' not found".format(template_path))
+        # Create a Jinja2 template from the read content
+        template = jinja2.Template(template_io.read().decode('utf-8'))
 
-		template = jinja2.Template(template_io.read().decode('utf-8'))
+        # Convert keys with dot notation into nested dictionaries
+        template_params = utils.create_nested_dict_from_dots_in_keys(jinja_variables)
```
Comment (ateska) [inline asabiris/formatter/jinja/service.py:72]: Is it tested?

I see that you prepared that `__getitem__` method.
But you are not applying that anywhere.

How do you apply that on `template_params`?

I guess you need to create some kind of wrapper class.

**Review by ateska — CHANGES_REQUESTED** [review]
Please, send me also the output of the test: jinja template with lower case, upper case and mixed case of the jinja parameter, the input dictionary and the output.

thanks.

**Comment by ateska** [comment]
Only case-insensitive dictionary wrapper for Jinja2 is needed here, due to https://github.com/TeskaLabs/asab-iris/pull/43

But b/c the code switched from tabs to spaces, it is difficult to isolate the actual implementation.

**Comment by ateska** [comment]
Please close this one and implement json loading in https://github.com/TeskaLabs/asab-iris/pull/43

**Comment by mithunbharadwaj** [comment]
Closing this MR. Solving here (https://github.com/TeskaLabs/asab-iris/pull/45)

**Pull request closed without merging**